sample-templates/dragonfly.conf

```diff
@@ -0,0 +1,13 @@
+# DragonFly 4.6 and later can boot with UEFI.  The Dragonfly installer works
+# with UEFI beginning at 4.8 
+uefi="yes"
+# The live CD has a serial console, but the installer requires graphics
+graphics="yes"
+cpu=1
+# 4GB of RAM is the minimum when using HAMMER.
```

Comment:
Looks like people run HAMMER on much lower RAM https://www.dragonflydigest.com/2012/01/02/8945.html

BUT looks like a lot of RAM is required for the installer CD to boot (it crashed with 1G)